gdnative-sys/build.rs

```diff
@@ -43,6 +43,13 @@ mod header_binding {
         Ok(directory)
     }
 
+    #[cfg(any(target_os = "android", target_os = "linux"))]
+    fn android_include_path() -> Option<String> {
+        let java_home = std::env::var("JAVA_HOME").expect("JAVA_HOME environment variable should be set");
```

Comment:
The `?` operator should be used here, otherwise this can only ever panic or return `Some(...)`, making the option return ineffective.

```suggestion
        let java_home = std::env::var("JAVA_HOME")?;
```

Reply:
I don't know what are you doing but I got this error using ? operator:

```
error[E0277]: `?` couldn't convert the error to `std::option::NoneError`
  --> /home/marcos.cabral/_w/godot-rust/gdnative-sys/build.rs:48:51
   |
48 |         let java_home = std::env::var("JAVA_HOME")?;
   |                                                   ^ the trait `std::convert::From<std::env::VarError>` is not implemented for `std::option::NoneError`
   |
   = note: the question mark operation (`?`) implicitly performs a conversion on the error value using the `From` trait
   = note: required by `std::convert::From::from`
```

Reply:
@mmacedoeu You can convert a `Result<T, _>` to an `Option<T>` using [`Result::ok`](https://doc.rust-lang.org/std/result/enum.Result.html#method.ok). You can then use the try operator (`?`) on it.

Reply:
Oh I am sorry, I assumed `std::env::var` would return an `Option<String>`. I updated the suggestion.